### PR TITLE
Perl6 raffler

### PR DESCRIPTION
**Pull request by jkva**
image

Perl6!

Perl6 takes a while to build and compile. The raffler itself is a one-liner, of course.

### CODE REVIEW

**Comment by jkva** [comment]
@lucasvanlierop travis faalt omdat de build zo lang duurt. Kan dat aangepast worden?

**Comment by jkva** [comment]
Now using perl6 Docker image - https://github.com/perl6/docker. Done in seconds 👍

File: jkva-rakudo-perl6/install_perl6.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```
Comment (lucasvanlierop) [inline jkva-rakudo-perl6/install_perl6.sh]: This can be removed I guess?

File: jkva-rakudo-perl6/run.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```
Comment (lucasvanlierop) [inline jkva-rakudo-perl6/run.sh]: You could also merge this into the docker `CMD`

Comment (jkva) [reply jkva-rakudo-perl6/run.sh]: Did that, and removed the p6 build script. Thanks!

**Comment by lucasvanlierop** [comment]
Thanks!! short

**Pull request merged**